docs-web/configuration/transportserver-resource.md

```diff
@@ -309,7 +309,7 @@ The [ConfigMap](/nginx-ingress-controller/configuration/global-configuration/con
 
 ## Limitations
 
-As of Release 1.7, the TransportServer resource is a preview feature. Currently, it comes with the following limitations:
-* When using TLS Passthrough, it is not possible to configure [Proxy Protocol](https://github.com/nginxinc/kubernetes-ingress/tree/v1.8.0/examples/proxy-protocol) for port 443 both for regular HTTPS and TLS Passthrough traffic.
+As of Release 1.8, the TransportServer resource is a preview feature. Currently, it comes with the following limitations:
```

Comment:
This should be reverted.

Think of it as saying "TransportServer is a preview(alpha) feature since it's release in 1.7." - This is still true/accurate in 1.8

```suggestion
As of Release 1.7, the TransportServer resource is a preview feature. Currently, it comes with the following limitations:
```

Open to rephrasing if you see fit, I've heard of others confused by this note too.